src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import AnnotatorContext from './AnnotatorContext';
+import { Action, AnnotationActionEvent, AnnotatorState, Status } from './types';
```

Comment:
I don't know should we define State type in another file?

Reply:
As discussed, the `AnnotatorState` happens to be the state of the hoc in this case, but any consumer of the context could use the same type as well